drivers/lcd/ssd1680.c

```diff
@@ -878,19 +896,30 @@ static void ssd1680_select(FAR struct ssd1680_dev_s *priv, bool cs)
     }
 }
 
-static void ssd1680_busy_wait(FAR struct ssd1680_dev_s *priv)
+static int ssd1680_busy_wait(FAR struct ssd1680_dev_s *priv)
 {
+  int maks_wait_ticks = 200;
   if ((priv->board_priv != NULL) && (priv->board_priv->check_busy != NULL))
     {
-      while (priv->board_priv->check_busy())
+      while (priv->board_priv->check_busy() && maks_wait_ticks > 0)
```

Comment:
```suggestion
      while (priv->board_priv->check_busy() && maks_wait_ticks-- > 0)
```
and remove code below.

Reply:
OK It is done